styles/variables-base.less

```diff
@@ -296,108 +305,107 @@
 @agate-sliderbutton-bg-transform: @agate-slider-bg-transform;
 @agate-sliderbutton-client-transform: @agate-slider-track-transform;
-@agate-sliderbutton-margin: inherit;
+@agate-sliderbutton-margin: 3px 24px;
 
 // Spinner
 // ---------------------------------------
-@agate-spinner-size: inherit;
+@agate-spinner-size: 52px;
 @agate-spinner-bg-margin: inherit;
-@agate-spinner-bg-padding: inherit;
-@agate-spinner-small-size: inherit;
+@agate-spinner-bg-padding: 9px;
+@agate-spinner-small-size: 36px;
 @agate-spinner-small-bg-margin: inherit;
-@agate-spinner-small-bg-padding: inherit;
-@agate-spinner-searching-width: inherit;
-@agate-spinner-searching-height: inherit;
-@agate-spinner-searching-border-radius: inherit;
-@agate-spinner-small-searching-width: inherit;
-@agate-spinner-small-searching-height: inherit;
-@agate-spinner-loading-height: inherit;
+@agate-spinner-small-bg-padding: 5px;
+@agate-spinner-searching-width: (@agate-spinner-size / 13);
+@agate-spinner-searching-height: (@agate-spinner-searching-width * 3);
+@agate-spinner-searching-border-radius: 50px;
+@agate-spinner-small-searching-width: (@agate-spinner-searching-width - 1);
+@agate-spinner-small-searching-height: (@agate-spinner-searching-height / 1.5);
+@agate-spinner-loading-height: (@agate-spinner-searching-height / 2);
 @agate-spinner-loading-width: @agate-spinner-loading-height;
-@agate-spinner-loading-border-radius: inherit;
-@agate-spinner-small-loading-height: inherit;
+@agate-spinner-loading-border-radius: 999px;
+@agate-spinner-small-loading-height: (@agate-spinner-small-searching-height / 2);
 @agate-spinner-small-loading-width: @agate-spinner-small-loading-height;
-@agate-spinner-searching-animation-duration: inherit;
-@agate-spinner-loading-animation-duration: inherit;
+@agate-spinner-searching-animation-duration: 1.2s;
+@agate-spinner-loading-animation-duration: 0.8s;
 
 // Spotlight
 // ---------------------------------------
 @agate-spotlight-outset: 12px;
 
 // Switch
 // ---------------------------------------
-@agate-switch-height: inherit;
-@agate-switch-width: inherit;
-@agate-switch-border-radius: inherit;
-@agate-switch-icon-scale: inherit;
+@agate-switch-height: 48px;
+@agate-switch-width: 96px;
+@agate-switch-border-radius: (@agate-switch-height / 2);
+@agate-switch-icon-scale: 2;
 @agate-switch-icon-font-family: @agate-icon-font-family;
-@agate-switch-icon-font-size: inherit;
-@agate-switch-icon-position-left: inherit;
-@agate-switch-selected-icon-position-left: inherit;
+@agate-switch-icon-font-size: (@agate-switch-height * @agate-switch-icon-scale);
+@agate-switch-icon-position-left: 0;
+@agate-switch-selected-icon-position-left: (@agate-switch-width - @agate-switch-height);
 
 // ToggleButton
 // ---------------------------------------
-@agate-togglebutton-indicator-position: inherit;
-@agate-togglebutton-indicator-border-width: inherit;
+@agate-togglebutton-indicator-position: auto 21px 9px 21px;
+@agate-togglebutton-indicator-border-width: 1px;
```

Comment:
I'm hesitent to assign specific border-widths to components since it's such a stylistic choice. If this is 0 by default, I think that will work out better for us.